src/patternfly/components/Alert/alert.scss

```diff
@@ -3,215 +3,166 @@
   --pf-c-alert--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-alert--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-alert--grid-template-columns: max-content 1fr max-content;
-  --pf-c-alert--grid-template-rows: 1fr auto;
+  --pf-c-alert--grid-template-rows: 1fr auto auto;
```

Comment:
What's the reasoning for `1fr` on the first row?

Reply:
Agree, I don't think it needs to be defined.